packages/lodestar/test/utils/state.ts

```diff
@@ -61,6 +66,12 @@ export function generateState(opts?: TestBeaconState, config = mainnetConfig): B
       epoch: GENESIS_EPOCH,
       root: ZERO_HASH,
     },
-    ...opts,
   };
+  treeBackedState = treeBackedState || config.types.BeaconState.tree.createValue(defaultState);
```

Comment:
Generating tree backed BeaconState is expensive (~4s in my environment) so I made a default tree backed state here. For the 1st test calling this function, I have to increase timeout for it. The down side is if we run 1 single test in local environment using `it.only` we may see this timeout issue if that test calls `generateState()`

Not sure if there're any better ways to resolve this.

Reply:
Maybe we can add a mocha `before` (outside of any `describe`) to call generateState
eg: `test/unit/setupState.ts` and the same for e2e

Reply:
thanks, that's a good idea

Reply:
Please use minimal preset for state, we cannot have mainnet state in unit tests

Reply:
so we should use minimal preset for unit tests and mainnet preset for e2e tests? Fyi, this file is shared for both unit and e2e tests.

Reply:
In my opinion we can use minimal in e2e as well. Only place for mainnet would be benchmark